lib/deploy.js

```diff
@@ -74,22 +77,40 @@ Please check configuration before continuing
 }
 
 function deploy (
-  logger /* : any */,
-  bundleUrl /* : string */,
+  bundleKey /* : string */,
+  accessToken /* : string */,
   stage /* : string */,
   serviceSettings /* : BMServerSettings */
 ) /* : Promise<void> */ {
-  // TODO Call Server CLI Service to deploy
-  logger.log(logSymbols.success, `
-Deployed:
-  Bundle Url: ${bundleUrl}
-
-To:
-  Deployment Url: ${serviceSettings.deploymentUrl}
-  Stage: ${stage}
-  Target Origin: https://example.com
-`)
-  return Promise.resolve()
+  const stopUpdates = logUpdates(() => `Deploying project - this may take several minutes...`)
+  return new Promise((resolve, reject) => {
+    request({
+      auth: {
+        bearer: accessToken
+      },
+      json: {
+        bundleBucket: serviceSettings.bucket,
+        bundleKey,
+        targetOrigin: 'https://example.com' // TEMP, needed for authorisation
```

Comment:
Yeah, I usually use `TODO` but this felt more like it would actually get removed by the end of day. meh